tests/shell/ShellCheckBearTest.py

```diff
@@ -17,7 +17,27 @@
 echo $X
 """
 
+valid_with_ignore_rules_file = """
+#!/usr/bin/env sh
+echo "Path is $PATH"
+cd /home
+for dir in */
+do
+  cd "$dir"
+  convert index.png index.jpg
+  cd ..
+done
+"""
+
+ShellCheckBearTest = verify_local_bear(
```

Comment:
@jayvdb I think i must break this line (and thus make the entire code block modified) 
because the invalid_files is too long. Any idea about this?